### PR TITLE
finish kata 4 get an article by its id

### PR DESCRIPTION
**Pull request by imaa04**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+       })
+      });
+});
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 sends an article by its id", () => {
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then((response) => {
+      console.log(response.body)
```
Comment (johnhutcheon) [inline __tests__/app.test.js:55]: Just make sure you remove any console logs before pushing work to github 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      });
+  });
+  test("GET:400 sends an appropriate status and error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/not-an-article")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
```
Comment (johnhutcheon) [inline __tests__/app.test.js:80]: Great testing! 🙌

File: app.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+//const endpoints = require('../endpoints.json')
```
Comment (johnhutcheon) [inline app.js]: Also remember to remove comments before pushing!

**Review by johnhutcheon — APPROVED** [review]
Great work Ima, all good to merge! 😄

File: app.js (hunk 3 above)
Comment (imaa04) [reply app.js]: Hi John, I wanted to ask if it's supposed to lose all of my work that I've done for kata 4 after i pull orgin main and switch to nc-news-5

**Pull request closed without merging**